scripts/jenkins/jenkins_jobs.ini

```diff
@@ -0,0 +1,7 @@
+[jenkins]
+user=USERNAME
+password=APITOKEN
+url=https://ci.suse.de/
```

Comment:
This is only for internal, but we need the same for external. Unfortunately you can't create 2 section. But maybe you can add a commented ```url=https://ci.opensuse.org/``` here?